plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryCaseInsensitiveMapping.java

```diff
@@ -46,6 +47,12 @@ public void initBigQueryExecutor()
         this.bigQuerySqlExecutor = new BigQuerySqlExecutor();
     }
 
+    @BeforeMethod(alwaysRun = true)
```

Comment:
`BeforeMethod` or `AfterClass(alwaysRun = true)`?

Reply:
@Before as the main issue is kill -9, so thanks to this we make sure that the environment is clean before every test. We already have cleanup in every test (try-final blocks etc.) and still it fails sometimes due to sudden termination. @After won't be invoked in case of kill -9

Reply:
Hmmm, can we instead use `@BeforeClass` in that case. That way even if kill -9 happens the next time CI starts the schemas will get cleaned up before any tests run. It also has the benefit of running once per class instead of per-method (which may cause us to run into rate-limits).

Reply:
Hm I guess for the issue at hand it will work and you are right about the possible rate limits issue